src/main/kotlin/g0101_0200/s0104_maximum_depth_of_binary_tree/readme.md

```diff
@@ -0,0 +1,26 @@
+104\. Maximum Depth of Binary Tree
```

Comment:
Please add implementation and test.

Reply:
We already have a unit test for this solution

Reply:
➕